src/components/giphy-toolbar-item.js

```diff
@@ -2,40 +2,42 @@
 import React from 'dom-chef'
 
 export default (
-  <details class="details-reset details-overlay toolbar-item select-menu select-menu-modal-right ghg-trigger">
-    <summary class="menu-target" aria-label="Insert a GIF" aria-haspopup="menu">
-      {'GIF'}
-    </summary>
-    <details-menu
-      class="select-menu-modal position-absolute right-0 ghg-modal"
-      style={{'z-index': 99, width: '480px', 'max-height': '410px'}}
-      role="menu"
-    >
-      <div class="select-menu-header d-flex">
-        <span class="select-menu-title flex-auto">Select a GIF</span>
-        <span class="ghg-powered-by-giphy" />
-      </div>
-      <tab-list>
-        <div class="select-menu-filters">
-          <div class="select-menu-text-filter">
-            <input
-              type="text"
-              class="form-control ghg-search-input"
-              placeholder="Search for a GIF…"
-              aria-label="Search for a GIF"
-              autofocus=""
-            />
-          </div>
+  <md-gif aria-label="GIF" id="md-gif" role="button" tabindex="-1" class="toolbar-item btn-octicon p-2 p-md-1 mx-1">
+    <details class="details-reset details-overlay toolbar-item select-menu select-menu-modal-right ghg-trigger">
```

Comment:
I don't think `md-gif` has any impact on the layout and we can instead just add the additional classes to the `details` element. 
```suggestion
    <details class="details-reset details-overlay toolbar-item select-menu select-menu-modal-right ghg-trigger btn-octicon p-2 p-md-1 mx-1">
```